content/reference/rest-api/engine-diagrams/create.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Engine Diagrams: create'
+weight: 10
+
+menu:
+  main:
+    name: 'create'
+    identifier: 'rest-api-engine-diagrams-create'
+    parent: 'rest-api-engine-diagrams'
+    pre: 'Creates or updates an engine diagram.'
+---
+
+Creates a new engine diagram if it doesn't exist yet or updates an engine diagram with a new version.
+
+# Request
+
+## HTTP Request
+
+```
+PUT https://cawemo.com/api/v1/engine-projects/:project-name/engine-diagrams
```

Comment:
Strictly speaking in terms of the rest API and maybe with the risk of missing some context here, I would like to ask why would we use `:project_name` since this should be a unique identifier?

To explain better where my question is coming from: Cawemo today does not enforce unique project/folder/file names in an organization/project or folder/project or folder.

Reply:
In today's meeting we agreed on using the project's id instead of the name as path variable.